docs/ops/memory/mem_tuning.md

```diff
@@ -63,13 +62,13 @@ will dictate the optimal memory configurations of your cluster.
 ### Heap state backend
```

Comment:
This whole section (`Configure memory for state backends`) seems to me only relevant to TMs. Maybe mention that as well, like what we do for `Configure memory for batch jobs`.

Reply:
I put it under `Configure memory for state backends`.